supabase/migrations/20230207011749_create_wallet_table.sql

```diff
@@ -2,5 +2,6 @@ CREATE TABLE wallets (
     user_name character varying(255) NOT NULL PRIMARY KEY,
     wallet_address character(42),
     created_at text,
-    updated_at text
-);
+    updated_at text,
+    multiplier text DEFAULT 1
+);
```

Comment:
You should NEVER change any migration file. You should create a new migration to add new fields so we can migrate it to the existing wallet table.

Reply:
https://github.com/ubiquity/bounty-bot/blob/development/supabase/README.md

Please take a look at the supabase readme

Reply:
> https://github.com/ubiquity/bounty-bot/blob/development/supabase/README.md
> 
> Please take a look at the supabase readme

Sorry bout that, So i should create a new migration with `supabase migration new`?

Reply:
@0xcodercrane